frontend/OBSApp.cpp

```diff
@@ -880,7 +930,7 @@ OBSApp::OBSApp(int &argc, char **argv, profiler_name_store_t *store)
 	snInt = new QSocketNotifier(sigintFd[1], QSocketNotifier::Read, this);
 	connect(snInt, &QSocketNotifier::activated, this, &OBSApp::ProcessSigInt);
 #else
-	connect(qApp, &QGuiApplication::commitDataRequest, this, &OBSApp::commitData);
+	connect(qApp, &QGuiApplication::commitDataRequest, this, &OBSApp::commitData, Qt::DirectConnection);
```

Comment:
This is also just catching `WM_QUERYENDSESSION` so it could be removed unless it does anything useful on other platforms.

Reply:
> This is also just catching `WM_QUERYENDSESSION` so it could be removed unless it does anything useful on other platforms.

Qt's documentation suggests that we ourselves should not try to exit in response to `commitDataRequest`, it should only save data. The session manager may try to `quit()` the application later in the process, depending how things respond.

This PR adjusted `commitData()` to only save data, I think at worst this is an extra safety.

Reply:
@derrod Is this feedback resolved?